### DISCUSSION TITLE
Export table to excel

### DISCUSSION THREAD

**Discussion by Margatos19** [JerBouma/FinanceDatabase#40] [Q&A] [answered]
When I export the complete table of stocks with all their information such as sectors, industries, etc., it does not include the symbols or ticker, is it normal?

**Comment by JerBouma** [comment, marked as answer]
Look into the exporting function of Pandas. Perhaps the index is not automatically exported. See https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.to_excel.html

This is by the way outside of the functionality of this database.
